### PR TITLE
.45ACP military pistols

### PR DESCRIPTION
**Pull request by tenmillimaster**
# Summary
```SUMMARY: Content "Adds two new .45 ACP military pistols"```

#### Purpose of change
Ever since I ~~axed the HK UCP~~ made sensible changes to the military pistols available, it has been mentioned that the number of Real guns in the military spawn group was quite low.

Even for players playing with fantasy guns enabled, removing the UCP meant that the rarer guns became more frequently spawning.
#### Describe the solution
Though my ultimate goal is to even more drastically update the pistol item groups, I've added two 'new' (ok they're old) .45 ACP pistols to the spawn tables for military pistols. One is a 1911 because the Marines are ridiculous, the other is the HK Mk 23 because no one was in their right minds in the ninety's. 

I've also added the Glock 19 to the military pistols on account of its use in the military for concealed carry. In addition, I've removed the suppressor from the USP 45 (no clue why it was spawning with one in the first place.

#### Describe alternatives you've considered
Adding the M9, the P226, and the HK P11 pistol to the tables/game. I'll get around to this when I have some better occurrence numbers.

### CODE REVIEW

File: data/json/items/gun/45.json [hunk 1]
```diff
@@ -92,6 +92,18 @@
     "magazine_well": 1,
     "magazines": [ [ "45", [ "m1911mag", "m1911bigmag" ] ] ]
   },
+  {
+    "id": "m1911_MEU",
+    "copy-from": "m1911",
+    "type": "GUN",
+    "name": "M45A1",
+    "description": "The M45A1 supplanted earlier M45 MEUSOC pistols in use by Force Recon elements of Marine Expeditionary Units.  Where the original M45 pistols were gutted M1911A1's hand-fitted by USMC armorers, the updated M45A1's are hardly different from any other commercial 1911 design save for their dual recoil springs.  Most were replaced in 2016 with Glock 19's due to logistics issues.",
```
Comment (Night-Pryanik) [inline data/json/items/gun/45.json]: Please remove any exact date from description.

Comment (ymber) [reply data/json/items/gun/45.json]: I think we're fine for dates of things that have actually happened. It's not a speculative date for something that might be affected by cata lore.

File: data/json/items/gun/45.json [hunk 2]
```diff
@@ -269,7 +281,17 @@
...
     "built_in_mods": [ "match_trigger" ],
-    "default_mods": [ "suppressor" ],
     "magazines": [ [ "45", [ "usp45mag" ] ] ]
+  },
+  {
+    "id": "mk23",
+    "copy-from": "usp_45",
+    "type": "GUN",
+    "name": "MK 23 MOD 0",
+    "description": "Jokingly referred to as \"The World's Only Crew-Served Pistol\", this massive pistol was designed as a primary weapon for select \"special operators\".  Its cumbersome nature, the introduction of the derivative HK USP series and the logistics of getting .45 ACP ammunition in theater doomed this behemoth to US SOCOM armories.  Like the USP, the Mk 23 is a remarkably reliable gun; someone could probably take out a nuclear equipped walking tank with this in their holster.",
```
Comment (LaVeyanFiend) [inline data/json/items/gun/45.json]: > someone could probably take out a nuclear equipped walking tank with this in their holster.

### !

**Comment by Throwaway-name** [comment]
>In addition, I've removed the suppressor from the USP 45 (no clue why it was spawning with one in the first place.

You could rename it ["USP Tactical"](https://en.wikipedia.org/wiki/Heckler_%26_Koch_USP#USP_Tactical) and make a unique suppressor for it.

**Comment by tenmillimaster** [comment]
Why?

The game would still need to cover HK45/USP 45's without them.  And a unique .45 ACP suppressor from b+t or Knight's cannot possibly be as common as more affordable ones from, say, liberty, silencerco, or gemtech.

**Comment by tenmillimaster** [comment]
Fixed the merge conflict with #32542.

**Pull request merged**